src/components/calcite-radio-button-group/calcite-radio-button-group.tsx

```diff
@@ -81,6 +90,11 @@ export class CalciteRadioButtonGroup {
 
   connectedCallback(): void {
     this.passPropsToRadioButtons();
+    this.mutationObserver?.observe(this.el, { childList: true, subtree: true });
```

Comment:
Do you need to watch for changes to radio-button children? I don't think it supports this, right?

Reply:
@jcfranco I don't think so. The props should only go down the dom right?

Reply:
> The props should only go down the dom right?

If I understand correctly, only one level, yes.